docs/content/releases.md

```diff
@@ -6,6 +6,69 @@ doctypes: ["concept"]
 toc: true
 ---
 
+## NGINX Ingress Controller 2.1.0
+
+06 Jan 2022
+
+OVERVIEW:
+
```

Comment:
style: can we have a list here - this would give this section better structure? Like here https://docs.nginx.com/nginx-ingress-controller/releases/#nginx-ingress-controller-200

Reply:
@pleshakov pushed a commit to create a list, not sure I did it right 😅  feel free to fix it

Reply:
@lucacome  looks allright! thanks!